Userland/Libraries/LibGUI/Widget.cpp

```diff
@@ -904,14 +904,8 @@ bool Widget::is_backmost() const
     return &parent->children().first() == this;
 }
 
-Action* Widget::action_for_key_event(KeyEvent const& event)
+Action* Widget::action_for_shortcut(Shortcut const& shortcut)
 {
-    Shortcut shortcut(event.modifiers(), (KeyCode)event.key());
-
-    if (!shortcut.is_valid()) {
-        return nullptr;
-    }
```

Comment:
Is it safe to remove this check?

Reply:
Ah good question, I'm guessing cause `Widget::action_for_shortcut` was only called from the more global `action_for_shortcut` helper (https://github.com/SerenityOS/serenity/pull/12291/files/6c995b7097a922613ece22e885ec6bce5ad584b3..bbc88d44dcf22f98f1f397271d06dafde3e59caa#diff-5fa4537fde3eb5f28c4f4c24dfad24517764cf45dcbb0e5854062cf9ce9b18e1R139), which does the same check before calling this, I removed it from within here and assumed callers were responsible for not passing dud shortcuts now. But could also add it back in here if it avoids us doing work when `Widget::action_for_shortcut` is called with an invalid shortcut?